app/models/iiif_print/pending_relationship.rb

```diff
@@ -3,5 +3,7 @@ class PendingRelationship < ApplicationRecord
     validates :parent_id, presence: true
     validates :child_title, presence: true
     validates :child_order, presence: true
+    validates :parent_model, presence: true
+    validates :child_model, presence: true
   end
```

Comment:
should be validate the presence of file_id too?

Reply:
Yes, I'll make that addition.

Reply:
Thanks, and feel free to merge when it looks good.